packages/native-stack/src/types.tsx

```diff
@@ -281,12 +283,36 @@ export type NativeStackNavigationOptions = {
   /**
    * Function which returns a React Element to display on the left side of the header.
    * This replaces the back button. See `headerBackVisible` to show the back button along side left element.
+   * Will be overriden by `headerLeftItems` on iOS.
    */
   headerLeft?: (props: NativeStackHeaderLeftProps) => React.ReactNode;
   /**
    * Function which returns a React Element to display on the right side of the header.
+   * Will be overriden by `headerRightItems` on iOS.
    */
   headerRight?: (props: NativeStackHeaderRightProps) => React.ReactNode;
+  /**
+   * Array of iOS native UIBarButtomItem or functions
+   * that returns a React Element to the left side of the header.
+   * Overrides `headerLeft`.
+   *
+   * @platform ios
+   */
+  headerLeftItems?: (
+    | NativeStackHeaderBarButtonItem
+    | NativeStackHeaderBarButtonItemWithCustomView<NativeStackHeaderLeftProps>
+  )[];
+  /**
+   * Array of iOS native UIBarButtomItem or functions
```

Comment:
The phrase "Array of iOS native UIBarButtomItem or functions" doesn't sound right, because you are neither specifying an array of `UIBarButtomItem` instances, nor functions.

Also I assume it should be `Button` and not `Buttom`

Something like "Array of items to display as UIBarButtonItem" maybe more accurate.